### PR TITLE
Add tests for accordion/accordion.html

### PR DESCRIPTION
**Pull request by spectranaut**

### CODE REVIEW

**Comment by spectranaut** [comment]
Hi @mcking65 -- can you loop in the accordion contributor? I'm not sure their username!

**Comment by mcking65** [comment]
@jessebeach, @tatermelon, @sh0ji, this is ready for review.

File: test/tests/accordion.js [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+
+  const buttons = await t.context.session.findElements(By.css(ex.buttonSelector));
+
+  for (let expandIndex = 0; expandIndex < buttons.length; expandIndex++) {
+
+    // Click a heading to expand the section
+    await buttons[expandIndex].click();
+
+    for (let index = 0; index < buttons.length; index++) {
+      let expandedValue = index === expandIndex ? 'true' : 'false';
```
Comment (jessebeach) [inline test/tests/accordion.js]: `const`

File: test/tests/accordion.js [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+
+  const buttons = await t.context.session.findElements(By.css(ex.buttonSelector));
+
+  for (let expandIndex = 0; expandIndex < buttons.length; expandIndex++) {
+
+    // Click a heading to expand the section
+    await buttons[expandIndex].click();
+
+    for (let index = 0; index < buttons.length; index++) {
+      let disabledValue = index === expandIndex ? 'true' : null;
```
Comment (jessebeach) [inline test/tests/accordion.js]: `const`

File: test/util/assertTabOrder.js [hunk 3]
```diff
@@ -9,7 +9,7 @@ const focusMatchesElement = async function (t, selector) {
       selector = arguments[0];
       return document.activeElement === document.querySelector(selector);
     }, selector);
-  }, 200);
+  }, t.context.WaitTime);
```
Comment (jessebeach) [inline test/util/assertTabOrder.js]: 👍

**Pull request merged**